sh/tests/greatest-of-all_test.sh

```diff
@@ -1,25 +1,31 @@
 #!/usr/bin/env bash
 
 # Unofficial Bash Strict Mode
-set -euo pipefail
+# set -euo pipefail
```

Comment:
If this is no longer used it should be removed not commented out

Reply:
done